GoMoney/Common/View/GMImageButton.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+
+class GMImageButton: UIView {
+    lazy var imageView: UIImageView = .build { _ in }
```

Comment:
```suggestion
    lazy var imageView: = UIImageView()
```

Reply:
I don't use `translatesAutoresizingMaskIntoConstraints = false` in `anchor extensions`
so I used `build` + `anchor` for constraint